password-strength-checker/enabled.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+module.exports = {
+  name: 'the password strength checker is enable',
+  hypothesis: 'the password strength checker will prevent users from signing up with insecure passwords.',
+  startDate: '2015-01-01',
+  subjectAttributes: ['uniqueUserId', 'isMetricsEnabled', 'forcePasswordCheck'],
+  independentVariables: ['passwordStrengthCheckEnabled'],
+  eligibilityFunction: function (subject) {
+    if (subject) {
+      if (subject.forcePasswordCheck === 'true') {
+        return true;
+      }
+
+      if (subject.forcePasswordCheck === 'false') {
+        return false;
+      }
+
+      if (subject.isMetricsEnabled) {
+        return true;
+      }
+    }
+
+    return false;
+  },
+  groupingFunction: function (subject) {
+    var trial = this.bernoulliTrial(0.1, subject.uniqueUserId);
```

Comment:
This would be 10% of 10%, or 1%. Sufficient, or should it just be open to everyone who has metrics collected? @vladikoff - thoughts?